opfiab/src/main/java/org/onepf/opfiab/OPFIab.java

```diff
@@ -101,7 +101,7 @@ static void unregister(@NonNull final Object subscriber) {
 
     /**
      * Posts event object for delivery to all subscribers.
-     * Intend to be used by {@link BillingProvider} implementations.
+     * Intend to be used by {@link BillingProvider} implementations only.
```

Comment:
That's not completely true. A lot of internal classes relay on this method, however it has `public` visibility because it also intended to be used by external modules.